packages/entities/entities-routes/src/composables/useToaster.ts

```diff
@@ -0,0 +1,12 @@
+import { onBeforeUnmount } from 'vue'
+import { ToastManager } from '@kong/kongponents'
+
+export default function useToaster() {
+  const toaster = new ToastManager()
```

Comment:
I noticed we previously removed the initialization of `ToastManager` from this repo in https://github.com/Kong/public-ui-components/issues/932, and decided to let the host apps manage the `ToastManager` instances.

What about passing the toast message as a string in an emitted event to the host apps? Below is an example from `@kong-ui-public/entities-routes`:

https://github.com/Kong/public-ui-components/blob/9349ae1cb9d67be2f0bf050c858346a4f383f489/packages/entities/entities-routes/src/components/RouteList.vue#L229